src/components/WheelSelect/index.js

```diff
@@ -67,12 +67,13 @@ const WheelSelect = ({ options, multipleSelect }) => {
                 className="wheel-select__container"
                 key={ index }
               >
-                <div className={ elementTextClass }>{ element.display }</div>
+                <div className={ elementTextClass } data-test-id="select-display">{ element.display }</div>
```

Comment:
How about naming this `wheel-select`?
```suggestion
                <div className={ elementTextClass } data-test-id="wheel-select">{ element.display }</div>
```